src/Util/FileUploader.php

```diff
@@ -94,10 +96,16 @@ public function combineChunks(Request $request): array
             fclose($chunk);
             unlink($chunksFolder . DIRECTORY_SEPARATOR . $i);
         }
-        // Success
-        fclose($targetFile);
         rmdir($chunksFolder);
+        $targetFileSize = StreamInfo::getFileSize(array('fileStream' => $targetFile));
+        fclose($targetFile);
+        if ($targetFileSize !== $fileSize) {
+            unlink($targetFileName);
+            rmdir($targetDirectory);
+            throw new HttpException(Response::HTTP_BAD_REQUEST, "File size does not match!\n Expected file size is:$fileSize, and got $targetFileSize!");
```

Comment:
This message is not getting displayed in PROD mode

Reply:
This is going to be a problem, I'm going to look into this deeper. I cannot return a response here.

Reply:
```suggestion
            throw new BadRequestHttpException("File size does not match!\n Expected file size is:$fileSize, and got $targetFileSize!");
```

Reply:
Maybe "Filesize doesn't match" is a good enough error message to the user